gallery/README.md

```diff
@@ -72,3 +72,28 @@ more details):
 ```
 flutter pub run grinder update-code-segments
 ```
+
+## Creating a new release (for Flutter org members)
+
+1. Bump the version number up in the `pubspec.yaml`. Use semantic versioning to determine 
+   which number to increment. For example `2.2.0+020200` should become `2.3.0+020300`.
+   
+2. Create a tag on the `master` branch in the form of `gallery-v2.3`.
```

Comment:
Do we want to add the git command for this?

Reply:
done